build.gradle.kts

```diff
@@ -5,6 +5,7 @@ plugins {
     id("net.researchgate.release") version "2.8.1"
     id("com.github.ben-manes.versions") version "0.38.0"
     id("maven-publish")
+    id("com.diffplug.spotless") version "7.0.0.BETA4"
```

Comment:
Can we have a stable version instead of Beta version ? 
Stable version to avoid potential issues down the line.

Reply:
We could roll back to [6.24.0](https://github.com/diffplug/spotless/releases/tag/gradle%2F6.24.0)